code/src/components/survey/Survey.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+import FirstName from "./FirstName";
+import LastName from "./LastName";
+import Gender from "./Gender";
+import Occupation from "./Occupation";
+import Age from "./Age";
+import Information from "./Information";
+
+const Survey = ({ onSubmit }) => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [gender, setGender] = useState();
+  const [occupation, setOccupation] = useState("");
+  const [age, setAge] = useState(18);
+  const [section, setSection] = useState(1);
+
+  const handleSubmitFirstName = ({ firstName }) => {
```

Comment:
Consider choosing to use a set of "back" and "next" buttons instead of "Submit" on each page. "Submit" feels like the end of a survey.